staging/src/k8s.io/pod-security-admission/metrics/metrics.go

```diff
@@ -16,9 +16,63 @@ limitations under the License.
 
 package metrics
 
+import (
+	admissionv1 "k8s.io/api/admission/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/pod-security-admission/admission"
+)
+
+const (
+	subsystem = "pod_security"
+)
+
+var SecutityEvaluation = metrics.NewCounterVec(
+	&metrics.CounterOpts{
+		Subsystem:      subsystem,
```

Comment:
subsystem should be the component (i.e. apiserver, kubelet, controller-manager, etc), perhaps namespace would be more appropriate here?